estimagic/inference/likelihood_covs.py

```diff
@@ -17,60 +17,60 @@ def cov_hessian(hessian):
     - The resulting hessian might not be positive definite and thus not invertible.
 
     Args:
-        hessian (np.array): 2d array hessian matrix of dimension (nparams, nparams)
+        hess (numpy.ndarray): 2d array hessian matrix of dimension (nparams, nparams)
 
     Returns:
-       hessian_matrix (np.array): 2d array covariance matrix (nparams, nparams)
+       numpy.ndarray: covariance matrix (nparams, nparams)
 
 
     Resources: Marno Verbeek - A guide to modern econometrics :cite:`Verbeek2008`
 
     """
-    info_matrix = -1 * (hessian)
+    info_matrix = -1 * hess
     cov_hes = robust_inverse(info_matrix, msg=INVALID_INFERENCE_MSG)
 
     return cov_hes
 
 
-def cov_jacobian(jacobian):
+def cov_jacobian(jac):
     """Covariance based on outer product of jacobian of loglikeobs.
 
     Args:
-        jacobian (np.array): 2d array jacobian matrix of dimension (nobs, nparams)
+        jac (numpy.ndarray): 2d array jacobian matrix of dimension (nobs, nparams)
 
     Returns:
-        jacobian_matrix (np.array): 2d array covariance matrix (nparams, nparams)
+        numpy.ndarray: covariance matrix of size (nparams, nparams)
 
 
     Resources: Marno Verbeek - A guide to modern econometrics.
 
     """
-    info_matrix = np.dot((jacobian.T), jacobian)
+    info_matrix = np.dot((jac.T), jac)
     cov_jac = robust_inverse(info_matrix, msg=INVALID_INFERENCE_MSG)
 
     return cov_jac
 
 
-def cov_sandwich(jacobian, hessian):
+def cov_sandwich(jac, hess):
     """Covariance of parameters based on HJJH dot product.
```

Comment:
This is the most robust version, isn't it? Would it make sense to rename this to cov_robust such that users see that it can make sense to use this method? 
Also once someone has the Hessian it is basically for free to calculate different versions of the standard errors. Is there any value in comparing these different standard errors?

Reply:
For MSM this is really the only one of practical relevance since nobody does optimal MSM. In the optimal case the two different versions are identical (up to floating point imprecision), in any other case using the optimal formula is invalid. But I like the idea of calling it `cov_robust`.